acceptance/tests/proxy_connection.rb

```diff
@@ -87,6 +87,7 @@
       assert(is_associated?(master, "pcp://#{agent}/agent"),
              "Agent #{agent} with PCP identity pcp://#{agent}/agent should be associated with pcp-broker")
     end
+    clear_squid_log(master)
```

Comment:
So that is why there were always two entries in the log! Yes this makes sense.

Reply:
Maybe we should also be verifying there's an entry here (for the PCP connection).

Reply:
I guess we do that as a separate test above.